tap_oracle/sync_strategies/helper.py

```diff
@@ -0,0 +1,12 @@
+import simplejson as json
+import sys
+from singer import RecordMessage
+
+
+def format_message(message,ensure_ascii=True):
+    return json.dumps(message.asdict(), use_decimal=True, ensure_ascii=ensure_ascii)
+
+
+def write_message(message, ensure_ascii=True):
+    sys.stdout.write(format_message(message, ensure_ascii=ensure_ascii) + '\n')
+    sys.stdout.flush()
```

Comment:
Any reason we are defining new methods than overwrting implementation in singer-python?

Reply:
https://github.com/singer-io/singer-python/blob/master/singer/messages.py#L225